services/web/server/tests/integration/test_exporter.py

```diff
@@ -335,15 +349,45 @@ async def test_import_export_import(
                 client, downloaded_file_path
             )
 
+    # duplicate newly imported project
+    url_duplicate = client.app.router["duplicate_project"].url_for(
+        project_id=imported_project_uuid
+    )
+    assert url_duplicate == URL(
+        API_PREFIX + f"/projects/{imported_project_uuid}:duplicate"
+    )
+    async with await client.post(url_duplicate, timeout=10) as duplicate_response:
+        assert duplicate_response.status == 200, await duplicate_response.text()
```

Comment:
FYI: check ``assert_status`` in [pytest_simcore helpers](https://github.com/ITISFoundation/osparc-simcore/blob/2a16137461/packages/pytest-simcore/src/pytest_simcore/helpers/utils_assert.py#L12)

Reply:
I see, but will not use in this case., will keep it like this, I find more readable.